__tests__/endpoints/getCostumes.test.js

```diff
@@ -0,0 +1,40 @@
+const {
+    getPool,
+    createTables, 
+    createCostume, 
+} = require('../../db/');
+
+const {
+    getBallroomGown
+} = require('../../db/utils');
+
+const {server} = require('../../app')
+const supertest = require('supertest');
+const request = supertest(server)
+
+
+let pool = getPool();
+
+// Double-check that correct database is being used. 
+if (pool.options.database !== 'costume_shop_db_test') {
+    throw new Error("Pool instance was not assigned testing database. Testing aborted. Be sure that NODE_ENV environment variable is set to 'test'.")
+}
+
+beforeEach( async () => {
+    await createTables(pool);
+    await createCostume(pool, getBallroomGown());;
```

Comment:
It's not necessary to do this in beforeEach - I would recommend moving it into your test-case?

Reply:
I created this thinking there would be more tests, but we ended up just doing the one and I forgot to take this code out of a `beforeEach`.